### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/feather/commit/52c008ace1c27996c6354532d1b47618e125272e#commitcomment-52106041

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/feather/commit/5c72d80c6a798a8d348ab3351ae406c3e3422d5a

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Guidelines for pull requests:
 
 Before creating an icon request, please search to see if someone has requested the icon already. If there is an open request, please add a :+1:.
 
-If the icon has not already been requested, [create an issue](https://github.com/colebemis/feather/issues/new?title=Icon%20Request:) with a title of `Icon request: <icon name>` and add as much information as possible.
+If the icon has not already been requested, [create an issue](https://github.com/feathericons/feather/issues/new?title=Icon%20Request:) with a title of `Icon request: <icon name>` and add as much information as possible.
```
Comment (jsoref) [inline CONTRIBUTING.md:26]: It looks like this repository moved to a new organization.

I've tried to only update the github items, and not paypal/similar.

File: bin/process-svg.js [hunk 2]
```diff
@@ -41,7 +41,7 @@ function optimize(svg) {
 }
 
 /**
- * Set default attibutes on SVG.
+ * Set default attributes on SVG.
```
Comment (jsoref) [inline bin/process-svg.js:44]: This is a harmless typo

File: src/tags.json [hunk 3]
```diff
@@ -102,7 +102,7 @@
   "key": ["password", "login", "authentication", "secure"],
   "layers": ["stack"],
   "layout": ["window", "webpage"],
-  "life-bouy": ["help", "life ring", "support"],
+  "life-buoy": ["help", "life ring", "support"],
```
Comment (jsoref) [inline src/tags.json:105]: This typo will probably have to live forever.

https://www.merriam-webster.com/dictionary/buoy
https://www.merriam-webster.com/dictionary/life%20buoy

> Definition of life buoy
> : a ring-shaped life preserver


https://www.merriam-webster.com/dictionary/bouy
> The word you've entered isn't in the dictionary. Click on a spelling suggestion below or try again using the search bar above.

Happy to drop this / file a bug / something else.

**Review by colebemis — APPROVED** [review]
Looks great. Thanks!

**Pull request merged**